From f4e43cb3efb3fefe3169a8697a96d8fb72d9e882 Mon Sep 17 00:00:00 2001
From: Roland Hieber <rhi@pengutronix.de>
Date: Tue, 15 Sep 2020 12:51:37 +0200
Subject: [PATCH] mci: bcm283x: depend on sdhci

The driver uses functions defined in sdhci.o, and will fail to build
otherwise:

    drivers/mci/mci-bcm2835.o: in function `bcm2835_mci_request':
    drivers/mci/mci-bcm2835.c:134: undefined reference to `sdhci_set_cmd_xfer_mode'
    arm-v5te-linux-gnueabi-ld: drivers/mci/mci-bcm2835.c:169: undefined reference to `sdhci_read_response'
    arm-v5te-linux-gnueabi-ld: drivers/mci/mci-bcm2835.c:175: undefined reference to `sdhci_transfer_data'

Forwarded: https://www.mail-archive.com/barebox@lists.infradead.org/msg36329.html
Signed-off-by: Roland Hieber <rhi@pengutronix.de>
---
 drivers/mci/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/mci/Kconfig b/drivers/mci/Kconfig
index f7dc5c508912..6fb1c200f43e 100644
--- a/drivers/mci/Kconfig
+++ b/drivers/mci/Kconfig
@@ -68,6 +68,7 @@ config MCI_S3C
 
 config MCI_BCM283X
 	bool "MCI support for BCM283X"
+	select MCI_SDHCI
 	depends on ARCH_BCM283X || COMPILE_TEST
 
 config MCI_BCM283X_SDHOST
-- 
2.28.0

